modules/compliance-supported-profiles.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/
+
+[id="compliance-supported-profiles_{context}"]
+= Compliance profiles
+
+The Compliance Operator provides the following compliance profiles:
+
+.Supported compliance profiles
```

Comment:
This table renders nicely. Are we going to add something that tells people how to find this information in the deployment itself? After reading the Jira ticket referenced in the commit message, it sounded like we wanted to provide some sort of statement describing how users can find this information themselves. Is my understanding correct that that information comes a few sections later?

https://deploy-preview-39324--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-understanding.html#compliance_profiles_understanding-compliance

Reply:
Hi @rhmdnd - The original request was to add this to Understanding section, but during a meeting, we decided to try it up a level in the doc. If the current positioning in the book is good,  I can add a link over to the Understanding section that explains how to find the profiles on your system.  Does that seem like a good approach? Alternately, I could just move the supported profile section under the Understanding  section, but I'm afraid that it gets obscured there.

Reply:
I'll ultimately defer to others, but I think the current positioning looks great. As a reader, I like having a table with the supported profiles up front. Your suggestion about adding a link to the Understanding section makes sense.